README.md

```diff
@@ -1172,6 +1172,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [pb](https://github.com/ptpb/pb) - Lightweight pastebin (and url shortener) built using flask. `GPL-3.0` `Python`
 - [pbnh](https://github.com/bhanderson/pbnh) - Pastebin inspired from project pb and hastebin, with an API and a SQL-based backend. `MIT` `Python`
 - [PrivateBin](https://privatebin.info/) - PrivateBin is a minimalist, opensource online pastebin/discussion board where the server has zero knowledge of hosted data. ([Demo](https://privatebin.net/), [Source Code](https://github.com/PrivateBin/PrivateBin)) `Zlib` `PHP`
+- [PurritoBin](https://github.com/PurritoBin/PurritoBin) - Ultra fast, minimalistic, encrypted command line paste-bin, where the server has no knowledge of the paste data. ([Demo](https://bsd.ac/)) `ISC` `C++/Javascript`
```

Comment:
- `C++` is enough as it is the only server-side requirement
- Please add `ISC` - [ISC License](https://spdx.org/licenses/ISC.html) to the [list of license](https://github.com/awesome-selfhosted/awesome-selfhosted#list-of-licenses)

Reply:
ISC is already in the list of licenses, not sure what you mean?

will change the language to C++ :smile_cat: